bool request

-----
sensor_msgs/LaserScan[] all_laser